distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -3133,6 +3133,259 @@ Supports "asc" and "desc" using colon as delimiter.
 
 The OpenSearch source does not provide a Connected Source interface.
 
+==== Confluence Source
+
+The Confluence source provides a Federated Source to retrieve pages/comments/attachments from a Confluence REST API  and turns the results into Metacards the system can use.
```

Comment:
Should "pages/comments/attachments" be "pages, comments, and attachments" instead to be more formal?